### PR TITLE
Ticket/24 active user list

### PR DESCRIPTION
**Pull request by sinanngok**

### CODE REVIEW

File: accounts/admin.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+    class Meta:
+        model = MyUser
+        fields = ('username', 'first_name', 'last_name')
+
+    def clean_password2(self):
+        # Check that the two password entries match
+        password1 = self.cleaned_data.get("password1")
+        password2 = self.cleaned_data.get("password2")
+        if password1 and password2 and password1 != password2:
```
Comment (efe) [inline accounts/admin.py]: Default of `required` attribute in `forms.Field` is `True`. Also, `forms.CharField` is inherited from `forms.Field`. And, `required` parameter isn't overrided in `forms.CharField`. So, `password1 != password2` is enough i guess.

File: foursquaresearch/utils.py [hunk 2]
```diff
@@ -46,8 +46,30 @@ def get_foursquare_results(location, what_to_look, venues, error_message, is_sea
...
-        history = PreviousSearch.objects.order_by('-created_date')
+        history = PreviousSearch.objects.order_by('-created_date')[:5]
 
     return (history)
+
+def get_all_logged_in_users():
+    # Query all non-expired sessions
+    # use timezone.now() instead of datetime.now() in latest versions of Django
+    sessions = Session.objects.filter(expire_date__gte=timezone.now())
+    uid_list = []
```
Comment (efe) [inline foursquaresearch/utils.py]: Please don't use shortened variable names.

File: accounts/middleware.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
+from .models import  MyUser as User
+from django.contrib.auth import authenticate
+from django.utils import timezone
+
+class SetLastVisitMiddleware(object):
+    def __init__(self, get_response):
```
Comment (efe) [inline accounts/middleware.py:6]: Inside of calling `__init__` and `__call__` methods, Can we set the `last_visit` datetime inside of `process_request` method?

**Review by efe — COMMENTED** [review]
It looks really nice!

However, Please don't just copy and paste the codes from Django documentation or StackOverflow. We should implement the code which satisfy only our needs. Also, have comment with your own words please.

File: accounts/middleware.py (hunk 3 above)
Comment (sinanngok) [reply accounts/middleware.py:6]: I didn't get what should I do here.

Comment (efe) [reply accounts/middleware.py:6]: There is a detailed tutorial for this. https://stackoverflow.com/a/32242074

**Pull request merged**